xDrip/Scenes/Home/Home/HomeInteractor.swift

```diff
@@ -113,6 +114,25 @@ final class HomeInteractor: HomeBusinessLogic, HomeDataStore {
         )
     }
     
+    private func setupUpdateTimer(with state: Home.SensorState) {
+        switch state {
+        case .started:
+            timer?.invalidate()
+            timer = nil
+        case .stopped:
+            if timer == nil {
+                timer = Timer.scheduledTimer(
+                    withTimeInterval: 60.0,
+                    repeats: true) { [weak self] _ in
+                        guard let self = self else { return }
+                        self.doLoad(request: Home.Load.Request())
```

Comment:
This is a bad approach because:
1. You're invoking a request from interactor to interactor. Interactor should only operate its private functions.
2. `Load` use case is designed to be called only once.

Instead of such logic, you should directly invoke the required private functions to update appropriate fields.